### DISCUSSION TITLE
Pycaret error message - Jinja2

### DISCUSSION THREAD

**Discussion by tliones** [pycaret/pycaret#2501] [General]
Hi - I keep receiving this error since Monday when trying to run in Colab - import pycaret.classification.  "ImportError: Missing optional dependency 'Jinja2'. DataFrame.style requires jinja2. Use pip or conda to install Jinja2."  I tried installing Jinja2 and refreshing the kernel etc.  no solution.  It started happening this Monday...was working all weekend.  Thanks!!

**Comment by ngupta23** [comment]
Please search in the closed issues. A solution was mentioned there.

**Reply by tliones** [reply]
Oh I see it now.  Sorry, I did a search and assumed it searched the closed issues as well.  Thanks so much.  It was driving me bonkers :).
